src/car/car.service.ts

```diff
@@ -2,6 +2,8 @@ import { BadRequestException, Injectable } from '@nestjs/common';
 import { ReservationService } from 'src/reservation/reservation.service';
 import { DatabaseService } from 'src/utils/database/database.service';
 import { DateQueryDto } from './dto/dateQuery.dto';
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const ObjectsToCsv = require('objects-to-csv');
```

Comment:
why don't you use `import` there?

Reply:
Using import for this module would result in the error show below:
`TypeError: objects_to_csv_1.default is not a constructor`
After some googling, people suggested that using const would work.
Do you have any solutions for this?

Reply:
```suggestion
import * as ObjectsToCsv from 'objects-to-csv';
```

Reply:
did you try this one?

Reply:
Yes absolutely.
This was the error:
![image](https://user-images.githubusercontent.com/124992083/220635917-757a534d-cf0c-4396-b0b6-753836128e68.png)

Reply:
![image](https://user-images.githubusercontent.com/124992083/220636882-3cc077e7-76a8-48b0-b420-74e62492928d.png)
![image](https://user-images.githubusercontent.com/124992083/220636939-8752d003-16f5-401d-b097-732b0bfd811c.png)

Reply:
okay